File: java/Processor.java
package latihan3.java;

public class Processor {

    private String name;
    private int price;

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    public Processor() {
    }

    public int getPrice() {
        return price;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: php/Processor.php
<?php


class Processor
{
    private $name, $price;

    public function __construct($name = "-", $price = 0)
    {
        $this->name = $name;
        $this->price = $price;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function getName()
    {
        return $this->name;
    }

    public function getPrice()
    {
        return $this->price;
    }
}